.github/workflows/CI.yml

```diff
@@ -24,6 +24,12 @@ jobs:
       # Clone the repo and checkout the commit for which the workflow was triggered
       - uses: actions/checkout@v2
 
+      - name: Test integrity of duplicate values
```

Comment:
It might be a good idea to be more specific on which duplicate values we are checking here

Reply:
I've added another test (for correctness of PWAServer in app.js) and have added comments to this script to make clearer what we are doing. Script still needs to be tested.